uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -1547,69 +1559,75 @@ sub CreateMRICandidates {
     ############################################################
     ### Standardize sex (DICOM uses M/F, DB uses Male/Female)
     ############################################################
-    my $this = shift;
-    my $query = '';
-    my ($subjectIDsref,$sex,$tarchiveInfo,$User,$centerID, $upload_id) = @_;
-    my ($message);
+    my $this  = shift;
+    my ($subjectIDsref, $tarchiveInfo, $User, $centerID, $upload_id) = @_;
+
+    my ($sex, $query, $message);
+    my $dbh    = $this->{dbhr};
+    my $pscID  = $subjectIDsref->{'PSCID'};
+    my $candID = $subjectIDsref->{'CandID'};
+
+
+    # return from the function if createCandidate config setting is not set
+    return if (!NeuroDB::DBI::getConfigSetting($dbh, 'createCandidates'));
+
+
+    # Check that no candidates with the same PSCID is already registered
+    if ($pscID ne 'scanner' && NeuroDB::MRI::subjectIDExists('PSCID', $pscID, $dbh)) {
 
+        $message = "ERROR: Cannot create candidate ($pscID, $candID) as "
```

Comment:
then check if the candidate's `PSCID` and `CandID` values stored in `$subjectIDsref` do not already refer to another candidate (there should not be two candidates with the same `PSCID` or `CandID`). If found a matching candidate, exit with proper error message.